src/Goerli-DssSpell.t.sol

```diff
@@ -493,4 +501,37 @@ contract DssSpellTest is GoerliDssSpellTestBase {
 
         assertEq(vest.rxd(1), WAD);
     }
+
+   function testTeleportFW() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        address router = addr.addr("MCD_ROUTER_TELEPORT_FW_A");
+        StarknetTeleportBridgeLike bridge = StarknetTeleportBridgeLike(addr.addr("STARKNET_TELEPORT_BRIDGE"));
+        address escrow = addr.addr("STARKNET_ESCROW");
+
+        bytes32 ilk = "TELEPORT-FW-A";
+        bytes23 domain = "ETH-GOER-A";
+
+        uint256 l2_teleport_gateway = 0x042b46146f0a377e0a028ed44bc1c0567196b8b96f3c7ab469e593ca497e2a83;
+
+        assertEq(bridge.escrow(), escrow);
+        assertEq(bridge.teleportRouter(), address(router));
+        assertEq(bridge.dai(), address(dai));
+        assertEq(bridge.l2DaiTeleportGateway(), l2_teleport_gateway);
+
```

Comment:
Can also check:
*  `assertEq(router.numDomains(), 4);` as done in 30/8 spell.
*  (TELEPORT_GATEWAY_STA).starkNet() value.